Test.Net35/ConnectionTests.cs

```diff
@@ -15,6 +16,31 @@ public void Init()
             Logger.Handlers += LoggingHandler;
         }
 
+        [Test()]
+        public void RetryErrorTestNet35()
+        {
+            Stopwatch watch = new Stopwatch();
+
+            // Set invalid host address and make timeout to 1s
+            var config = new Config().SetAsync(false);
+            config.SetHost("https://api.segment.com");
```

Comment:
This is not very future proof as `api.segment.com` could be a valid address in the future (if we decide to put it up). I think it's better if we use something obviously invalid here.